arwenmandos/execScenario.go

```diff
@@ -165,3 +169,61 @@ func (ae *ArwenTestExecutor) ExecuteTxStep(step *mj.TxStep) (*vmi.VMOutput, erro
 
 	return output, nil
 }
+
+// PutNewAccount Puts a new account in world account map. Overwrites.
+func (ae *ArwenTestExecutor) PutNewAccount(mandosAccount *mj.Account) error {
+	worldAccount, err := convertAccount(mandosAccount, ae.World)
+	if err != nil {
+		return err
+	}
+	err = validateSetStateAccount(mandosAccount, worldAccount)
+	if err != nil {
+		return err
+	}
+
+	ae.World.AcctMap.PutAccount(worldAccount)
+	return nil
+}
+
+// UpdateAccount Updates an account in world account map.
+func (ae *ArwenTestExecutor) UpdateAccount(mandosAccount *mj.Account) error {
+	worldAccount, err := convertAccount(mandosAccount, ae.World)
+	if err != nil {
+		return err
+	}
+	err = validateSetStateAccount(mandosAccount, worldAccount)
+	if err != nil {
+		return err
+	}
+
+	existingAccount := ae.World.AcctMap.GetAccount(mandosAccount.Address.Value)
```

Comment:
Duplicate code as in the method above. Merge them in one method and make the update check there.